docs/devel/networking.md

```diff
@@ -56,6 +56,8 @@ the following operations, in this order:
 The virt-launcher is an untrusted component of KubeVirt (since it wraps the
 libvirt process that will run third party workloads). As a result, it must be
 run with as little privileges as required.
+In particular, it should not assume having `CAP_NET_ADMIN` or `CAP_NET_RAW`
+capabilites.
```

Comment:
What is so special about these 2 ? Do we want to highlight that until recently they were required and now are not ?

Do we want to indicate that `CAP_NET_BIND_SERVICE` is needed ?

Reply:
We know from history that they are very tempting to use. I didn't know about `CAP_NET_BIND_SERVICE` - can you elaborate? What needs it?

Reply:
In is need to bind to low ports, like port 67 of the dhcp server.

Reply:
I think these two are "heavier" and prone to abuse. E.g. with NET_RAW one can build packets and send them on the network attacking others anonymously.
With something like `CAP_NET_BIND_SERVICE`, one can just occupy well known services ports. These is one security issue with it: it allows someone with access to the pod to raise a non-raw socket (e.g. DHCP) that serves the outer network.. something that seems pretty bad.

Reply:
I find it very weird that we list the capabilities that are **not** required but omit the one that is.

Reply:
> raise a non-raw socket (e.g. DHCP) that serves the outer network.. something that seems pretty bad.

to me, this paragraph is about do's and don'ts for the KubeVirt network developers. The important thing is that we must not reintroduce the risky CAPs; if someone drops `CAP_NET_BIND_SERVICE`, I don't mind.

Can you suggest another text that would make things clearer, @maiqueb ?

Reply:
This change seems misleading to me, since it highlights 2 capabilities in particular, which, imho, could mislead the reader into thinking any other capability can be added. 

Furthermore, it's not like a developer can assume those capabilities exist: once they try to use those, the whole pod will explode. They would have to update the virt-launcher pod's template, explicitly adding those. Which would be automatically rejected by any reviewer.

IMO, If you really want to highlight "dont's" for the kubevirt network developers, say that they shouldn't add **any** capabilities to the virt-launcher pod template. 

/hold so that the bot doesn't merge this.